server/src/lib.rs

```diff
@@ -92,7 +96,12 @@ pub async fn start(config: ServerConfig) -> Result<()> {
                         }
                     });
                 }
-                Err(e) => log::error!("Failed to accept the request: {:?}", e),
+                Ok(Err(e)) => log::error!("Failed to accept the request: {:?}", e),
+                Err(_) => {
+                    // timeout, cleanup old connections
+                    let mut manager = manager.lock().await;
+                    manager.cleanup().await;
```

Comment:
For Websocket, the longer time connection is expected, if this changes applied, does it mean the websocket connection will be closed after 60 seconds?

Reply:
No, the client won't be removed if at least one WebSocket is still actually using the tunnel.

But I can make it to whatever duration you prefer regardless.